packages/core/src/queries/user.ts

```diff
@@ -99,10 +113,11 @@ const buildUserConditions = (search: Search, hideAdminUser: boolean) => {
     Users.fields.name,
   ];
 
-  if (hideAdminUser) {
-    // Cannot use \`= any()\` here since we didn't find the Slonik way to do so. Consider replacing Slonik.
+  if (excludeUserIds.length > 0) {
+    // FIXME @sijie temp solution to filter out admin users,
```

Comment:
how about we just find the id of `Admin` role first, and just pass role ids here. then we just need one-level join and the query should be fine

Reply:
It will still need 2-level: `users` and `users_roles`